inst/templates/scorecard-template.Rmd

```diff
@@ -40,6 +40,11 @@ knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE,
 library(dplyr)
 library(flextable)
 set_flextable_defaults(font.color = "#333333", border.color = "#999999", padding = 4, fonts_ignore = TRUE)
+
+scorecard_type <- params[["pkg_scores"]][["scorecard_type"]]
+if (is.null(scorecard_type)) {
+  stop("pkg_scores unexpectedly does not have 'scorecard_type' field")
+}
```

Comment:
I took a look at the `format_traceability_matrix()` changes too; specifically noting how the report changes for `R` packages` versus the external ones ('command' vs 'exported function'). 

Was there any consideration about more clearly denoting the `scorecard_type` within the report?

Reply:
> Was there any consideration about more clearly denoting the `scorecard_type` within the report?

I'm open to suggestions.  I personally don't the value/need to show that at the report level.

Reply:
>I'm open to suggestions

I was thinking of maybe modifying one of the first few sentences to just more explicitly state the software type, but I think @seth127 would be better suited to chime in here. Was mostly wondering if it might be desirable for QA folks.

Reply:
> modifying one of the first few sentences to just more explicitly state the software type

Somewhat related to that: I did look through the PDF for parts that seemed too tied to the R context, but I didn't spot anything.  Please let me know if you do.

Reply:
This is an interesting question. At this point, I don't see a clear right-or-wrong answer, but I'll continue to mull it over. 

In absence of a reason to add it, I would say to leave it as is.